truss/tests/templates/server/common/test_logging.py

```diff
@@ -0,0 +1,23 @@
+from truss.templates.server.common.logging import CustomJsonFormatter
+
+
+def test_log_parsing():
+    formatter = CustomJsonFormatter()
+    log_record = {
+        "message": '172.17.0.1:52518 - "POST /v1/models/model%3Apredict HTTP/1.1" 200'
+    }
+
+    formatter.process_log_record(log_record)
+    assert log_record["message"] == "POST /predict 200 OK"
+
+    log_record = {
+        "message": '172.17.0.1:52518 - "POST /v1/models/model%3Apredict_binary HTTP/1.1" 500'
+    }
+    formatter.process_log_record(log_record)
+    assert log_record["message"] == "POST /predict_binary 500 INTERNAL_ERROR"
+
+    log_record = {
+        "message": '172.17.0.1:52518 - "POST /v1/models/model%3Apredict HTTP/1.1" 503'
+    }
+    formatter.process_log_record(log_record)
+    assert log_record["message"] == "POST /predict 503 SERVICE_UNAVAILABLE"
```

Comment:
Can you add another test for a log string that doesn't match this? I am worried about the negative case, where the logger receives a message that doesn't map to an API call, which is what this is targetting

Reply:
+1 to adding test coverage for negative case to make sure that no formatting is applied to log records that don't match the pattern